Update awayfull.html

Conman1161 opened the pull request:
Add various features:

- Adds dynamic changing of social media listings via Dashboard program (does not resize div if missing profile, default 2 profiles) REQUIRES ADDITION OF 4 VARIABLES TO DASHBOARD PROGRAM. NAMES FOUND IN IMAGE IN DISCORD CHANNEL
- Social media listings now swipe upward to appear rather than being a permanent fixture. Aims to make everything consistent in appearance rather than having certain elements swipe in and some elements being always visible.
- The "VS" separator between upcoming matches now swipes outward from the middle rather than being permanent, mimicking the teams that slide out immediately next to it.

gitagogaming commented on HTML Packs/DEFAULT HTML/html/awayfull.html:
```diff
@@ -370,6 +413,27 @@
         /* content:"\f1e8" twitch logo*/
     }
 
+    .bluesky:before {
```
Could we keep an entry for twitch here as well rather than replacing it?  
This way other users still have it as a choice. 

Also curious if you had looked for the X logo?

looks like this might be a potential 'hack' for it 
https://www.briancoords.com/twitter-icon-font-awesome-4/

Conman1161 replied:
Twitch should still be present? I didn't adjust Twitch's logo CSS at all, it should still be at line 402 in this PR.

I had not looked into the X logo, but I can certainly add that in an edit.

gitagogaming reviewed (commented):
Looks good but did have a comment/request on the logos available for socials

Conman1161 commented:
Added in the newest version of the Font Awesome font files and added in a few additional CSS rules for logos. More information about each change can be found in each commit.
